src/doc/man/cargo-info.md

```diff
@@ -0,0 +1,63 @@
+# cargo-info(1)
+
+## NAME
+
+cargo-info --- Display information about a package in the registry. Default registry is crates.io
+
+## SYNOPSIS
+
+`cargo info` [_options_] _crate_@_version_
+
+## DESCRIPTION
+
+This command displays information about a package in the registry. It fetches data from the package's Cargo.toml file
+and presents it in a user-friendly format.
```

Comment:
Should we highlight that this is not machine-readable nor programmable format?

Reply:
Sounds good to me. Let me rephrase it.

Reply:
Updated.